xml/admin_gui_dashboard.xml

```diff
@@ -2287,6 +2287,89 @@ Oct 04 07:48:53 doc-ses6min4 systemd[1]: Started Ceph rbd mirror daemon.
 &prompt.cephuser;ceph dashboard create-self-signed-cert
 </screen>
    </sect3>
+   <sect3 xml:id="self-sign-certificates-openssl">
+     <title>Self-signed or Trusted Third-party Certificate with OpenSSL</title>
+     <para>
+       OpenSSL is an open-source command line tool that is commonly used
+       to generate private keys, create CSRs, install your SSL/TLS
+       certificate, and identify certificate information. The following
+       instructions illustrate to generate a self-signed or trusted third-party
+       certificate using OpenSSL:
```

Comment:
consider using entity for 'OpenSSL' to unify its capitalization (and introduce it if it does not exist)